app/scripts/vendor/env-test.js

```diff
@@ -81,4 +81,11 @@
   if (isStyleAllowed(style)) {
     docElement.className += (' ' + style);
   }
+
+  /**
+   * Check if the user is signing in to Sync on Firefox for iOS. User agent sniff. Yuck.
+   */
+  if (/FxiOS/.test(navigator.userAgent) && parseQueryParams().service === 'sync') {
```

Comment:
Not sure if we can use the 'isSync()' helper here https://github.com/mozilla/fxa-content-server/blob/3562e8f5c788b0b5661fd99965dedd8fe5832f40/app/scripts/models/reliers/relier.js#L70

Or if we add some util functions somewhere for the gross 'isAndroid' and 'isIOS' detection functions.